packages/query/src/index.ts

```diff
@@ -968,6 +968,10 @@ export class LiveQuery implements WithTx, Client {
     result: LookupData<T>
   ): Promise<void> {
     for (const key in lookup._id) {
+      if ((doc as any)[key] === undefined || (doc as any)[key] === 0) {
```

Comment:
This is not always true, we could still have $lookups without a collection with numbers. @BykhovDenis please take a look on this change.